tests/integration/components/lt-body-test.js

```diff
@@ -33,7 +33,7 @@ test('row selection - enable or disable', function(assert) {
 
   this.render(hbs `{{lt-body table=table sharedOptions=sharedOptions canSelect=canSelect}}`);
 
-  let row = this.$('tr:first');
+  let row = this.$('tr:not(.lt-scaffolding-row):first');
```

Comment:
In the original state of this PR with `enableScaffolding` defaulting to `true`, this was necessary. It isn't *anymore*, but I precautiously still included this commit. This way the tests will continue to work, even if we decide to set `enableScaffolding` by default in the future, or when we copy and paste these cases to write some new `enableScaffolding` specific tests.

If you feel that this makes the tests harder to comprehend / maintain, I have absolutely not objections to cancel out this commit. :slightly_smiling_face: